mapactionpy_arcmap/arcmap_runner.py

```diff
@@ -126,28 +140,79 @@ def getOrientation(countryName):
         print("Error: Could not derive country extent from " + url)
         print("Exiting.")
         sys.exit(1)
```

Comment:
Again I think `sys.exit()` is too draconian within the `getOrientation()` function. I would raise an exception here and handle it near where the commandline args are parsed. What happens if `https://nominatim.openstreetmap.org/search` is not available? Can the automation proceed?